### ISSUE TITLE
Unable to load charts elements

### ISSUE THREAD

**Issue by Mushaqdeen-J** [rajeshvaya/angular-material-widget-engine#4]
Hi,
     Thanks for this great plugin, but I am not able to load charts into this widget. Is it possible to load? I am using angular js charts.

**Comment by rajeshvaya** [comment]
@Mushaqdeen-J Yes, it is possible to load charts within the widget. [One of the demo](https://rajeshvaya.github.io/angular-material-widget-engine/demo/simple_dashboard/)  files have Google charts added in the widget engine. You can include any angular directive within the widget.

The documentation about various configuration is in https://rajeshvaya.github.io/angular-material-widget-engine/

If you want to see the source code of the demo which includes charts, you can see the [gh-pages branch](https://github.com/rajeshvaya/angular-material-widget-engine/tree/gh-pages)

Hope this helps

**Comment by Mushaqdeen-J** [comment]
Hi, thanks for your response. How can I use controller scope element in my content .html file?
